### ISSUE TITLE
Documentation URLs now broken

### ISSUE THREAD

**Issue by jnothman** [explosion/spaCy#1505]
Congratulations on the spaCy 2.0 release! In the move to 2.0 all links (and search engine indexes) to 1.x docs are broken. I'm not convinced that helps anyone.

It might be a good idea to introduce HTTP redirects to the legacy documents, or introduce redirection pages which help the user navigate to the legacy document, while using the canonical URL metadata to tell search engines to look at the new docs.

**Comment by ines** [comment]
That's weird – maybe something went wrong when porting over the redirects from alpha.spacy.io over to spacy.io. There's a whole set of pretty sophisticated redirects that should cover pretty much all old docs pages and even link anchors and it worked fine in the alpha.

Will have a look!

**Comment by jnothman** [comment]
Ha. I should probably back up my statements with an example. Now to find
what I landed on before...

**Comment by jnothman** [comment]
Sorry, I think it might have been a mistaken diagnosis on my part. Google sent me to https://spacy.io/docs/usage/processing-text which is a 404. Not sure if there are other issues.

**Comment by jnothman** [comment]
Well, yes, when I ask google for "spacy nlp" it includes that URL and also https://spacy.io/usage/deep-learning which are broken.

**Comment by ines** [comment]
No worries, I have a list of all pages here as well (which I used to write all the specific redirects).  I think part of it might have been Cloudflare's cache.

Thanks for the additional links btw! Looks like we definitely forgot the `processing-text` one. The `deep-learning` one was supposed to be on the new site, but we moved stuff around at the last minute, so this should definitely be included in the redirects as well.
